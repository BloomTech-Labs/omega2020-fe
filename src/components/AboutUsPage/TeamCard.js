import React from 'react';
import Box from '@material-ui/core/Box';
import Avatar from '@material-ui/core/Avatar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import GitHubIcon from '@material-ui/icons/GitHub';
import LinkedInIcon from '@material-ui/icons/LinkedIn';
import { blueGrey } from '@material-ui/core/colors';
import { makeStyles } from '@material-ui/core/styles';
import ClickAwayListener from '@material-ui/core/ClickAwayListener';
import Grow from '@material-ui/core/Grow';
import Popper from '@material-ui/core/Popper';
import MenuList from '@material-ui/core/MenuList';

const TeamCard = (props) => {
  const classes = useStyles();

  const { name, image, title, linkedin, github } = props.member || {};

  const [open, setOpen] = React.useState(false);
  const anchorRef = React.useRef(null);

  const handleToggle = () => {
    setOpen((prevOpen) => !prevOpen);
  };

  const handleClose = (event) => {
    if (anchorRef.current && anchorRef.current.contains(event.target)) {
      return;
    }

    setOpen(false);
  };

  function handleListKeyDown(event) {
    if (event.key === 'Tab') {
      event.preventDefault();
      setOpen(false);
    }
  }

  // return focus to the button when we transitioned from !open -> open
  const prevOpen = React.useRef(open);
  React.useEffect(() => {
    if (prevOpen.current === true && open === false) {
      anchorRef.current.focus();
    }

    prevOpen.current = open;
  }, [open]);

  return (

    <Box className={classes.root}>
      <Avatar
        alt={name}
        src={image}
        className={classes.large}
        ref={anchorRef}
        aria-controls={open ? 'menu-list-grow' : undefined}
        aria-haspopup='true'
        onClick={handleToggle}
        style={{ cursor: 'pointer' }}
      />
      <br />
      <Box>
        <Popper
          placement='center'
          open={open}
          anchorEl={anchorRef.current}
          role={undefined}
          transition
          disablePortal
          style={{ zIndex: 1 }}
        >
          {({ TransitionProps, placement }) => (
            <Grow
              {...TransitionProps}
              style={{
                transformOrigin:
                  placement === 'bottom-end' ? 'bottom-end' : 'bottom-end',
              }}
            >
              <Box className={classes.info}>
                <ClickAwayListener onClickAway={handleClose}>
                  <MenuList 
                    autoFocusItem={open}
                    id='menu-list-grow'
                    onKeyDown={handleListKeyDown}
                    data-testid='container'
                  >
                    <Typography variant='subtitle2' data-testid='name'>{name}</Typography>
                    <Typography variant='caption' data-testid='title'>{title}</Typography>
                    <Box className={classes.icons}>
                      <IconButton href={github} data-testid='github'>
                        <GitHubIcon
                          style={{ fontSize: 25, color: blueGrey[300] }}
                        />
                      </IconButton>
                      <IconButton href={linkedin} data-testid='linkedin'>
                        <LinkedInIcon
                          style={{ fontSize: 25, color: blueGrey[300] }}
                        />
                      </IconButton>
                    </Box>
                  </MenuList>
                </ClickAwayListener>
              </Box>
            </Grow>
          )}
        </Popper>
      </Box>
    </Box>
  );
};

const useStyles = makeStyles((theme) => ({
  root: {
    display: 'flex',
    flexFlow: 'row wrap',
    '& > *': {
      margin: theme.spacing(2),
    },
  },
  large: {
    width: theme.spacing(7),
    height: theme.spacing(7),
  },
  info: {
    padding: 20,
  },
}));

export default TeamCard;
