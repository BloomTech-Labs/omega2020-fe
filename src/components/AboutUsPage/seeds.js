const TeamV1 = [
  {
    version: '1',
    name: 'Hira Khan',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-UG8U1EMQC-23707ef54cc3-512',
    title: 'Data Scientist',
    linkedin: 'https://www.linkedin.com/in/hira-shahid-991b1583/',
    github: 'Hira63S',
  },
  {
    version: '1',
    name: 'Rob Hamilton',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-ULJ9DTDKL-246bfe8730a9-512',
    title: 'Data Scientist',
    linkedin: 'https://www.linkedin.com/in/rob1ham/',
    github: 'Rob1Ham',
  },
  {
    version: '1',
    name: 'Sheila Eichenberger',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-UHZCNQY3Y-a25cb56e80d9-512',
    title: 'Web Developer',
    linkedin: 'N/A',
    github: 'EpiceneDev',
  },

  {
    version: '1',
    name: 'Connor R Sullivan',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-ULVE6L3TJ-5b9eaad31d15-512',
    title: 'Web Developer',
    linkedin: 'n/a',
    github: 'crsullivan',
  },
  {
    version: '1',
    name: 'Gabriel Romero',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-UEFKRMPCP-4d6c51049ae9-512',
    title: 'UX Designer',
    linkedin: 'www.linkedin.com/in/gabe-romero-design',
    github: 'n/a',
  },
  {
    version: '1',
    name: 'Levi Simpson',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-UDCJB03RD-0e87fb5029cc-512',
    title: 'Team Lead',
    linkedin: 'n/a',
    github: 'Levi-Huynh',
  },
  {
    version: '1',
    name: 'Rudy Enriquez',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-UD2LGR18A-7c6caf1a9c21-512',
    title: 'Data Scientist',
    linkedin: 'n/a',
    github: 'RNEnriquez',
  },
  {
    version: '1',
    name: 'Johana Luna',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-ULJAKAQ66-88214d54e62a-512',
    title: 'Data Scientist',
    linkedin: 'https://www.linkedin.com/in/johanaluna/?locale=en_US',
    github: 'johanaluna',
  },
  {
    version: '1',
    name: 'Akak Almaz',
    image: 'https://ca.slack-edge.com/T4JUEB3ME-ULK6ULQ0J-b2b2713290b3-512',
    title: 'Web Developer',
    linkedin: 'n/a',
    github: 'Aakak',
  },
];

const TeamV2 = [
  // WEB
  {
    version: '2',
    name: 'Jessica Dosseh',
    image:
      'https://avatars2.githubusercontent.com/u/24831198?s=460&u=4e2d7a78a8d4ba798589a59075bb98cd1b0e7e26&v=4',
    title: 'Web Developer',
    linkedin: 'https://www.linkedin.com/in/jessicadosseh/',
    github: 'https://github.com/JessicaDosseh',
  },
  {
    version: '2',
    name: 'Tara Sherman',
    image:
      'https://avatars1.githubusercontent.com/u/44125075?s=460&u=0d892d66ffa3913439ffd4badbc0d4875f5ecd32&v=4',
    title: 'Web Developer',
    linkedin: 'https://www.linkedin.com/in/tarasherman/',
    github: 'https://github.com/taraSherman',
  },
  {
    version: '2',
    name: 'Carlos Turcios',
    image: 'https://ca.slack-edge.com/ESZCHB482-W0138D436D6-e0838d1d105e-512',
    title: 'Web Developer',
    linkedin: 'https://www.linkedin.com/in/carlos-turcios-b5051055/',
    github: 'https://github.com/CarlosETurcios',
  },
  {
    version: '2',
    name: 'Alexis Anderson',
    image: 'https://ca.slack-edge.com/ESZCHB482-W012BRNRD9C-a2e830df8d7f-512',
    title: 'Web Developer',
    linkedin: 'https://www.linkedin.com/in/alexis-anderson-530795196/',
    github: 'https://github.com/aanderson9313',
  },
  {
    version: '2',
    name: 'Vincent Adeniji',
    image: 'https://ca.slack-edge.com/ESZCHB482-W012BRP432S-5ca2303ff84a-512',
    title: 'Web Developer',
    linkedin: 'https://www.linkedin.com/in/vincent-adeniji-363008116/',
    github: 'https://github.com/TOSINNIJIS1',
  },

  // DS
  {
    version: '2',
    name: 'Stephen Plainte',
    image: 'https://ca.slack-edge.com/ESZCHB482-W012QNTLJ2Y-eded290f44d6-512',
    title: 'Data Scientist',
    linkedin: 'https://www.linkedin.com/in/stephenplainte/',
    github: 'https://github.com/FuriouStyles',
  },
  {
    version: '2',
    name: 'Ivan Santos',
    image: 'https://ca.slack-edge.com/ESZCHB482-W0138D56KRN-8828cd79a091-512',
    title: 'Data Scientist',
    linkedin: 'https://www.linkedin.com/in/ivan-santos-a3688624/',
    github: 'https://github.com/Skantastico',
  },
  {
    version: '2',
    name: 'Brandon Bruce',
    image: 'https://ca.slack-edge.com/ESZCHB482-W012H6NP04V-449729fd2101-512',
    title: 'Data Scientist',
    linkedin: 'https://www.linkedin.com/in/brandon-bruce-1538a21a8/',
    github: 'https://github.com/brucebra000',
  },

  // DS + TL 1
  {
    version: '2',
    name: 'Marvin Davila',
    image: 'https://ca.slack-edge.com/ESZCHB482-W012X6PSYDP-ec35dbf6dfdb-512',
    title: 'Team Lead',
    linkedin: 'https://www.linkedin.com/in/marvin-davila/',
    github: 'https://github.com/malexmad',
  },
  // WEB + TL 2
  {
    version: '2',
    name: 'Richard Lawson',
    image: 'https://ca.slack-edge.com/ESZCHB482-W012X6T9KT3-443632831451-512',
    title: 'Team Lead',
    linkedin: 'https://www.linkedin.com/in/lawsonarichard/',
    github: 'https://github.com/lawsonarichard',
  },
];

export { TeamV1, TeamV2 };
