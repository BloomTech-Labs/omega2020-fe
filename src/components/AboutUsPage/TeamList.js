import React from 'react';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
import TeamCard from './TeamCard';
import { TeamV1, TeamV2 } from './seeds';

const TeamVersion1 = () => {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <Typography>Version 1 Team</Typography>
      <div data-testid='member' className={classes.team}>
        {TeamV1.map((member) => (
          <TeamCard data-testid='members' member={member} />
        ))}
      </div>
    </div>
  );
};

const TeamVersion2 = () => {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      {/* <Typography>Version 2 Team</Typography> */}
      <Typography>Active Team</Typography>
      <div className={classes.team}>
        {TeamV2.map((member) => (
          <TeamCard member={member} />
        ))}
      </div>
    </div>
  );
};

const useStyles = makeStyles((theme) => ({
  root: {
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignContent: 'center',
    textAlign: 'center',
    marginTop: 50,
    marginBottom: 50,
    '& > *': {
      margin: theme.spacing(3),
    },
  },
  team: {
    display: 'flex',
    flexFlow: 'row wrap',
    justifyContent: 'center',
  },
}));

export { TeamVersion1, TeamVersion2 };
