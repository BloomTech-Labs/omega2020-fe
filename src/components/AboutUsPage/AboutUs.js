import React from 'react';
import Box from '@material-ui/core/Box';
import Skeleton from '@material-ui/lab/Skeleton';
import CardHeader from '@material-ui/core/CardHeader';
import Typography from '@material-ui/core/Typography';
import AccessibilityNewIcon from '@material-ui/icons/AccessibilityNew';
import GroupIcon from '@material-ui/icons/Group';
import { makeStyles } from '@material-ui/core/styles';
import { TeamVersion1, TeamVersion2 } from './TeamList';

const AboutUs = (props) => {
  const classes = useStyles();
  const { loading = false } = props;

  return (
    <Box className={classes.root}>
      <Box className={classes.banner}>
        <CardHeader
          title={
            loading ? (
              <Skeleton className={classes.title} width={400} height={100} />
            ) : (
              <Box>
                <Typography variant='h4' data-testid='head1'>
                  About Sudomega
                </Typography>
                <TeamVersion2 />
              </Box>
            )
          }
        />
      </Box>

      <Box className={classes.mainContainer}>
        <Box className={classes.leftContainer}>
          <Box className={classes.container}>
            <CardHeader
              title={
                loading ? (
                  <Skeleton
                    className={classes.title}
                    width={400}
                    height={100}
                  />
                ) : (
                  <Box>
                    <Box className={classes.img}>
                      <AccessibilityNewIcon style={{ fontSize: 100 }} />
                    </Box>
                    <br />
                    <Typography variant={'h5'} data-testid='head2'>
                      Our Story
                    </Typography>
                  </Box>
                )
              }
            />
            <CardHeader
              subheader={
                loading ? (
                  <Skeleton width={400} height={400} />
                ) : (
                  <Typography>
                    Sudomega started as a Lambda School Labs project, meaning we
                    were given an idea and told to run with it! Currently we are
                    the second team to work on Sudomega, previously known as
                    Omega2020. Our goal is to bring Sudoku to everyone. We want
                    to make it easy and fun to learn while still challenging
                    more experienced users.
                  </Typography>
                )
              }
            />
          </Box>
        </Box>

        <Box className={classes.rightContainer}>
          <Box className={classes.container}>
            <CardHeader
              title={
                loading ? (
                  <Skeleton
                    className={classes.title}
                    width={400}
                    height={100}
                  />
                ) : (
                  <Box>
                    <Box className={classes.img}>
                      <GroupIcon style={{ fontSize: 100 }} />
                    </Box>
                    <br />
                    <Typography variant={'h5'} data-testid='head2'>
                      About Our Teams
                    </Typography>
                  </Box>
                )
              }
            />
            <CardHeader
              subheader={
                loading ? (
                  <Skeleton width={400} height={400} />
                ) : (
                  <Typography data-testid='subheader'>
                    An overview of the founding team and core contributors to
                    Sudomega. Teams are composed of Web Developers, Ux
                    Designers, and Data Scientists. Feel free to click on each
                    members picture to learn more about them!
                  </Typography>
                )
              }
            />
          </Box>
        </Box>
      </Box>

      <hr width='100%' />
      <TeamVersion1 />
      <br />
      <br />
    </Box>
  );
};

const useStyles = makeStyles(() => ({
  root: {
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignContent: 'center',
    alignItems: 'center',
    textAlign: 'center',
    marginTop: 150,
  },
  mainContainer: {
    width: '100%',
    display: 'flex',
    flexFlow: 'row wrap',
    justifyContent: 'space-evenly',
    marginBottom: 150,
  },
  leftContainer: {
    width: '600px',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    padding: 30,
    textAlign: 'center',
  },
  rightContainer: {
    width: '600px',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    padding: 30,
    textAlign: 'center',
  },
}));

export default AboutUs;
