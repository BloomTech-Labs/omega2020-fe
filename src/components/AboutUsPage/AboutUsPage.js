import React from 'react';
import Box from '@material-ui/core/Box';
import Container from '@material-ui/core/Container';
import AboutUs from './AboutUs';

const AboutUsPage = () => {
  return (
    <Box>
      <Container>
        <AboutUs />
      </Container>
    </Box>
  );
};

export default AboutUsPage;
