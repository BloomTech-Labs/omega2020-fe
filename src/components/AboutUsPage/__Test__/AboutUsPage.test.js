import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import  { configure, shallow } from 'enzyme';
import '@testing-library/jest-dom/extend-expect';
import AboutUsPage from '../AboutUsPage';
import Adapter from 'enzyme-adapter-react-16';
import { Box, Container } from '@material-ui/core/';
import { createMount } from '@material-ui/core/test-utils';


configure({ adapter: new Adapter() });

it('Makes a snapshot of the component', () => {
    const aboutUsPage = renderer.create(<AboutUsPage />).toJSON()
    expect(aboutUsPage).toMatchSnapshot();
})


it('renders the material ui elements', () => {

    const mount =  createMount();
    const wrapper = mount(<AboutUsPage /> )

    expect(wrapper.find(Box)).toHaveLength(51)
    expect(wrapper.find(Container)).toHaveLength(1)


})