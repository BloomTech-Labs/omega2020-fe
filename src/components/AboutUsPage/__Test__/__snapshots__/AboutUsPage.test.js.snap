// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Makes a snapshot of the component 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      className="MuiBox-root MuiBox-root-6 makeStyles-root-2"
    >
      <div
        className="MuiBox-root MuiBox-root-7"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          >
            <span
              className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
            >
              <div
                className="MuiBox-root MuiBox-root-8"
              >
                <h4
                  className="MuiTypography-root MuiTypography-h4"
                  data-testid="head1"
                >
                  About Sudomega
                </h4>
                <div
                  className="makeStyles-root-9"
                >
                  <p
                    className="MuiTypography-root MuiTypography-body1"
                  >
                    Active Team
                  </p>
                  <div
                    className="makeStyles-team-10"
                  >
                    <div
                      className="MuiBox-root MuiBox-root-14 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Jessica Dosseh"
                          className="MuiAvatar-img"
                          src="https://avatars2.githubusercontent.com/u/24831198?s=460&u=4e2d7a78a8d4ba798589a59075bb98cd1b0e7e26&v=4"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-15"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-16 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Tara Sherman"
                          className="MuiAvatar-img"
                          src="https://avatars1.githubusercontent.com/u/44125075?s=460&u=0d892d66ffa3913439ffd4badbc0d4875f5ecd32&v=4"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-17"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-18 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Carlos Turcios"
                          className="MuiAvatar-img"
                          src="https://ca.slack-edge.com/ESZCHB482-W0138D436D6-e0838d1d105e-512"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-19"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-20 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Alexis Anderson"
                          className="MuiAvatar-img"
                          src="https://ca.slack-edge.com/ESZCHB482-W012BRNRD9C-a2e830df8d7f-512"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-21"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-22 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Vincent Adeniji"
                          className="MuiAvatar-img"
                          src="https://avatars2.githubusercontent.com/u/54925113?s=460&v=4"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-23"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-24 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Stephen Plainte"
                          className="MuiAvatar-img"
                          src="https://ca.slack-edge.com/ESZCHB482-W012QNTLJ2Y-eded290f44d6-512"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-25"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-26 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Ivan Santos"
                          className="MuiAvatar-img"
                          src="https://ca.slack-edge.com/ESZCHB482-W0138D56KRN-8828cd79a091-512"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-27"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-28 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Brandon Bruce"
                          className="MuiAvatar-img"
                          src="https://ca.slack-edge.com/ESZCHB482-W012H6NP04V-449729fd2101-512"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-29"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-30 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Marvin Davila"
                          className="MuiAvatar-img"
                          src="https://ca.slack-edge.com/ESZCHB482-W012X6PSYDP-ec35dbf6dfdb-512"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-31"
                      />
                    </div>
                    <div
                      className="MuiBox-root MuiBox-root-32 makeStyles-root-11"
                    >
                      <div
                        aria-haspopup="true"
                        className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
                        onClick={[Function]}
                        style={
                          Object {
                            "cursor": "pointer",
                          }
                        }
                      >
                        <img
                          alt="Richard Lawson"
                          className="MuiAvatar-img"
                          src="https://ca.slack-edge.com/ESZCHB482-W012X6T9KT3-443632831451-512"
                        />
                      </div>
                      <br />
                      <div
                        className="MuiBox-root MuiBox-root-33"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </span>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-34 makeStyles-mainContainer-3"
      >
        <div
          className="MuiBox-root MuiBox-root-35 makeStyles-leftContainer-4"
        >
          <div
            className="MuiBox-root MuiBox-root-36"
          >
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <span
                  className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                >
                  <div
                    className="MuiBox-root MuiBox-root-37"
                  >
                    <div
                      className="MuiBox-root MuiBox-root-38"
                    >
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root"
                        focusable="false"
                        style={
                          Object {
                            "fontSize": 100,
                          }
                        }
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M20.5 6c-2.61.7-5.67 1-8.5 1s-5.89-.3-8.5-1L3 8c1.86.5 4 .83 6 1v13h2v-6h2v6h2V9c2-.17 4.14-.5 6-1l-.5-2zM12 6c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2z"
                        />
                      </svg>
                    </div>
                    <br />
                    <h5
                      className="MuiTypography-root MuiTypography-h5"
                      data-testid="head2"
                    >
                      Our Story
                    </h5>
                  </div>
                </span>
              </div>
            </div>
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <p
                  className="MuiTypography-root MuiTypography-body1"
                >
                  Sudomega started as a Lambda School Labs project, meaning we were given an idea and told to run with it! Currently we are the second team to work on Sudomega, previously known as Omega2020. Our goal is to bring Sudoku to everyone. We want to make it easy and fun to learn while still challenging more experienced users.
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="MuiBox-root MuiBox-root-39 makeStyles-rightContainer-5"
        >
          <div
            className="MuiBox-root MuiBox-root-40"
          >
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <span
                  className="MuiTypography-root MuiCardHeader-title MuiTypography-h5 MuiTypography-displayBlock"
                >
                  <div
                    className="MuiBox-root MuiBox-root-41"
                  >
                    <div
                      className="MuiBox-root MuiBox-root-42"
                    >
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root"
                        focusable="false"
                        style={
                          Object {
                            "fontSize": 100,
                          }
                        }
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M16 11c1.66 0 2.99-1.34 2.99-3S17.66 5 16 5c-1.66 0-3 1.34-3 3s1.34 3 3 3zm-8 0c1.66 0 2.99-1.34 2.99-3S9.66 5 8 5C6.34 5 5 6.34 5 8s1.34 3 3 3zm0 2c-2.33 0-7 1.17-7 3.5V19h14v-2.5c0-2.33-4.67-3.5-7-3.5zm8 0c-.29 0-.62.02-.97.05 1.16.84 1.97 1.97 1.97 3.45V19h6v-2.5c0-2.33-4.67-3.5-7-3.5z"
                        />
                      </svg>
                    </div>
                    <br />
                    <h5
                      className="MuiTypography-root MuiTypography-h5"
                      data-testid="head2"
                    >
                      About Our Teams
                    </h5>
                  </div>
                </span>
              </div>
            </div>
            <div
              className="MuiCardHeader-root"
            >
              <div
                className="MuiCardHeader-content"
              >
                <p
                  className="MuiTypography-root MuiTypography-body1"
                  data-testid="subheader"
                >
                  An overview of the founding team and core contributors to Sudomega. Teams are composed of Web Developers, Ux Designers, and Data Scientists. Feel free to click on each members picture to learn more about them!
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
      <hr
        width="100%"
      />
      <div
        className="makeStyles-root-9"
      >
        <p
          className="MuiTypography-root MuiTypography-body1"
        >
          Version 1 Team
        </p>
        <div
          className="makeStyles-team-10"
          data-testid="member"
        >
          <div
            className="MuiBox-root MuiBox-root-43 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Hira Khan"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-UG8U1EMQC-23707ef54cc3-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-44"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-45 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Rob Hamilton"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-ULJ9DTDKL-246bfe8730a9-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-46"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-47 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Sheila Eichenberger"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-UHZCNQY3Y-a25cb56e80d9-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-48"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-49 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Connor R Sullivan"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-ULVE6L3TJ-5b9eaad31d15-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-50"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-51 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Gabriel Romero"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-UEFKRMPCP-4d6c51049ae9-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-52"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-53 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Levi Simpson"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-UDCJB03RD-0e87fb5029cc-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-54"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-55 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Rudy Enriquez"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-UD2LGR18A-7c6caf1a9c21-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-56"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-57 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Johana Luna"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-ULJAKAQ66-88214d54e62a-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-58"
            />
          </div>
          <div
            className="MuiBox-root MuiBox-root-59 makeStyles-root-11"
          >
            <div
              aria-haspopup="true"
              className="MuiAvatar-root MuiAvatar-circle makeStyles-large-12"
              onClick={[Function]}
              style={
                Object {
                  "cursor": "pointer",
                }
              }
            >
              <img
                alt="Akak Almaz"
                className="MuiAvatar-img"
                src="https://ca.slack-edge.com/T4JUEB3ME-ULK6ULQ0J-b2b2713290b3-512"
              />
            </div>
            <br />
            <div
              className="MuiBox-root MuiBox-root-60"
            />
          </div>
        </div>
      </div>
      <br />
      <br />
    </div>
  </div>
</div>
`;
